pkg/client/clientset_generated/clientset/fake/register.go

```diff
@@ -46,11 +46,13 @@ func init() {
 //   )
 //
 //   kclientset, _ := kubernetes.NewForConfig(c)
-//   aggregatorclientsetscheme.AddToScheme(clientsetscheme.Scheme)
+//   _ = aggregatorclientsetscheme.AddToScheme(clientsetscheme.Scheme)
 //
 // After this, RawExtensions in Kubernetes types will serialize kube-aggregator types
```

Comment:
This entire comment seems redundant...

Reply:
This is generated code. It is possible that it's redundant.